Add docs on custom admin (#346)
Opster/opensearch-k8s-operator: docs/userguide/main.md

        secretName: secret-name
```

## Custom Admin User

> In order to create your cluster with other than default `admin:admin` user you will have to add walk through those steps: 

Reviewer: ```suggestion
In order to create your cluster with an adminuser different from the default `admin:admin` you will have to walk through the following steps: 
```

First you will have to create an `admin-credentials-secret` secret with your admin user configuration:
```yaml
apiVersion: v1
kind: Secret
metadata:
  name: admin-credentials-secret
type: Opaque
data:
  # admin
  username: YWRtaW4=
  # admin123
  password: YWRtaW4xMjM=
```

> Second you will have to create your own `securityconfig-secret` secret (take a look about the at opensearch-operator/examples/securityconfig-secret.yaml for an example).

Reviewer: ```suggestion
Second you will have to create your own `securityconfig-secret` secret (take a look at `opensearch-operator/examples/securityconfig-secret.yaml` for an example).
```
